data/state/export_test.go

```diff
@@ -25,6 +25,10 @@ func (adb *AccountsDB) GetObsoleteHashes() map[string][][]byte {
 	return adb.obsoleteDataTrieHashes
 }
 
+func GetCode(account baseAccountHandler) []byte {
```

Comment:
GetCodeHash?

Reply:
this is an export function only. some stuff is mocked for tests only.

Reply:
Yes it is, but it returns a code hash and not the code, the name is misleading also for tests :)